### ISSUE TITLE
DLL Load failed when "import opensim“ in Python with OpenSim 4.2

### ISSUE THREAD

**Issue by Allwillcome** [opensim-org/opensim-core#3048]
DLL Load failed：
![image](https://user-images.githubusercontent.com/35840764/127089151-d82aa144-55ff-4c44-b6e4-7e5cf10d92ab.png)
![image](https://user-images.githubusercontent.com/35840764/127089185-0c727c5e-fe5d-4c26-a5e8-90fe69e11d01.png)

**Basic Infomation：**

windows

OS name: Microsoft Windows 10 profession

OS version: 10.0.18362 Build 18362

Python==3.7 （in Anaconda Virtual Environment）

Numpy ==1.19.2

Scipy == 1.7.0

Opensim = 4.2（Use GUI）
![image](https://user-images.githubusercontent.com/35840764/127089233-72aae69a-03cd-49ff-8eaf-7fc53381592e.png)

I followed the tutorial step by step, but it didn't work.

My python version is 64 bits, and I've added Opensim to the environment variable.

In addition, there is a contradiction about tutorials:

> Edit your Path environment variable to include <OPENSIM_INSTALL_DIRECTORY>\bin is in system variables Path, where OPENSIM_INSTALL_DIRECTORY is the directory where you installed the current version of OpenSim (e.g. C:\OpenSim 4.0). 

> Edit your PYTHONPATH environment variable to include <OPENSIM_INSTALL_DIRECTORY>\sdk\Python is in system variable PYTHONPATH, where OPENSIM_INSTALL_DIRECTORY is the directory where you installed the current version of OpenSim (e.g. C:\OpenSim 4.0). Delete any other OpenSim Path entries

Besides, Don't I need carry out Special instructions for Python 3.8+ on Windows?

The Function of "OpenSim's DLLs are not on your PATH. In a command window, you could achieve this with a command like "set PATH=C:\OpenSim 4.0\bin;%PATH%""  And add Path through GUI in windows is the same function?

**Comment by Allwillcome** [comment]
I

**Comment by aymanhab** [comment]
Can you clarify what was the issue, in case somebody run into it in the future. Thanks

**Comment by Allwillcome** [comment]
> Can you clarify what was the issue, in case somebody runs into it in the future. Thanks

Thank you for your attention，I have described my issue，When I try to import OpenSim，the program will report an error：DlLL failed loaded. Later on, I found that I mistakenly added the environment variable path information. When I opened the environment variable, I clicked New instead of clicking on the path and then clicking New. While fixing the bug，I still can't import OpenSim.
Finally, I give up the OpenSim 4.2, and try to opensim4.3 beta, And I successfully import opensim package.
If you are still confused about my question, I would be happy to provide more information.
